yaptide/batch/fluka_string_templates.py

```diff
@@ -0,0 +1,101 @@
+SUBMIT_FLUKA: str = """#!/bin/bash
+OUT=`mktemp`
+module load fluka
+
+ROOT_DIR={root_dir}
+cd $ROOT_DIR
+mkdir -p $ROOT_DIR/workspaces/task_{{0001..{n_tasks}}}
+mkdir -p $ROOT_DIR/input
+
+
+INPUT_DIR=$ROOT_DIR/input
+ARRAY_SCRIPT=$ROOT_DIR/array_script.sh
+COLLECT_SCRIPT=$ROOT_DIR/collect_script.sh
+
+unzip -d $INPUT_DIR $ROOT_DIR/input.zip
+rm $ROOT_DIR/input.zip
+
+FLUKA_CMD="sbatch --array=1-{n_tasks} {array_options} --parsable $ARRAY_SCRIPT > $OUT"
+eval $FLUKA_CMD
+JOB_ID=`cat $OUT | cut -d ";" -f 1`
+echo "Job id: $JOB_ID"
+
+if [ -n "$JOB_ID" ] ; then
+    COLLECT_CMD="sbatch --dependency=afterany:$JOB_ID {collect_options} --parsable $COLLECT_SCRIPT > $OUT"
+    eval $COLLECT_CMD
+    COLLECT_ID=`cat $OUT | cut -d ";" -f 1`
+    echo "Collect id: $COLLECT_ID"
+fi
+"""
+
+COLLECT_FLUKA_BASH: str = """#!/bin/bash
+{collect_header}
+ROOT_DIR={root_dir}
+python3 $ROOT_DIR/simulation_data_sender.py --sim_id={sim_id} --update_key={update_key} \\
```

Comment:
that should go just before a call to convertmc, 
what if `mkdir -p $OUTPUT_DIRECTORY` fails ?